assets/js/hocs/with-default-attributes.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * External dependencies
+ */
+import { createHigherOrderComponent } from '@wordpress/compose';
+const { getBlockType } = wp.blocks;
+
+/**
+ *  withDefaultAttributes
+ *
+ *  Fixes a bug where block attribute "default values" are not saved in serialized data.
+ *  Extend the default BlockListBlock component and apply default values allowing
+ *  Gutenberg to serialize the settings (normally ignored).
+ *
+ *  @see		https://github.com/WordPress/gutenberg/issues/7342#issuecomment-435371583
+ *
+ *  @param	object BlockListBlock The BlockListBlock element.
+ */
+const withDefaultAttributes = createHigherOrderComponent(
```

Comment:
I think this should be moved outside of the `js/hocs` folder. The hoc here has a _very specific_ use case so it'd be nice to keep it separate for the more general hoc usages.  probably would be good to keep it in `editor/hocs` (staying connected with the filters there) maybe?

Reply:
What about in the filter js file itself since it's so specific?

Reply:
Done 👍